mino/minogrpc/mod.go

```diff
@@ -116,3 +132,45 @@ func (m Minogrpc) MakeRPC(name string, h mino.Handler) (mino.RPC, error) {
 
 	return rpc, nil
 }
+
+// players implements mino.Players{}
+type players struct {
```

Comment:
It is not intended to be implemented at that level. Mino should use an implementation from a higher level module like skipchain.